public/docs/_examples/attribute-directives/ts/src/app/app.component.html

```diff
@@ -1,5 +1,4 @@
-<!-- #docregion -->
-<!-- #docregion v2 -->
+<!-- #docregion v2, -->
```

Comment:
I know that you _can combine_ named and _the unnamed region_ in a single line like this. I'm not sure that it helps the reader. Those trailing commas are hard to pick up. But let's forge ahead with this change.